Sources/JWTKit/EdDSA/EdDSA.swift

```diff
@@ -19,7 +19,7 @@ extension EdDSA {
     ///
     /// In JWT, EdDSA public keys are represented as a single x-coordinate and are used for verifying signatures.
     /// Currently, only the ``EdDSACurve/ed25519`` curve is supported.
-    public struct PublicKey: EdDSAKey {
+    public struct PublicKey: EdDSAKey, Equatable {
```

Comment:
Out of curiosity, why can't we also make them Hashable?

Reply:
We could if there's a use case for it. I wouldn't have made them `Equatable` either if 
1. We didn't need it in the tests
2. RSA and ECDSA weren't also already `Equatable`

Reply:
I think all keys should conform to `Hashable` because… they are